R/imageutils.R

```diff
@@ -1,17 +1,25 @@
 startPNG <- function(filename, width, height, res, ...) {
+  args <- rlang::list2(filename=filename, width=width, height=height, res=res, ...)
+
   # If quartz is available, use png() (which will default to quartz).
   # Otherwise, if the Cairo package is installed, use CairoPNG().
   # Finally, if neither quartz nor Cairo, use png().
   if (capabilities("aqua")) {
     pngfun <- grDevices::png
+  } else if ((getOption('shiny.useragg') %OR% TRUE) &&
+             nchar(system.file(package = "ragg"))) {
+    pngfun <- ragg::agg_png
+    args$background <- args$background %OR% args$bg
+    args$bg <- NULL
```

Comment:
Especially when it comes to text rendering quality, `quartz()` seems preferable to `ragg::agg_png()`. 

When it comes to `ragg` vs Cairo, the answer is [it depends](https://ragg.r-lib.org/articles/ragg_quality.html) on what you're interested in, so I think it makes sense to introduce a new `shiny.useragg` option (maybe this should default to `FALSE` for now?)